src/main/java/com/actiontech/dble/net/mysql/CommandPacket.java

```diff
@@ -112,6 +112,53 @@ public void write(MySQLConnection c) {
         }
     }
 
+    public void writeBigPackage(MySQLConnection c, int size) {
+        ByteBuffer buffer = null;
+        int remain = 0;
+        try {
+            boolean isFirst = true;
+            while (size >= MySQLPacket.MAX_PACKET_SIZE) {
+                buffer = c.allocate(MySQLPacket.MAX_PACKET_SIZE + MySQLPacket.PACKET_HEADER_SIZE);
+                size -= MySQLPacket.MAX_PACKET_SIZE;
+                BufferUtil.writeUB3(buffer, MySQLPacket.MAX_PACKET_SIZE);
+                buffer.put(packetId++);
+                remain = writeBody(buffer, isFirst, remain);
+                c.write(buffer);
+                isFirst = false;
+            }
+            buffer = c.allocate(size + MySQLPacket.PACKET_HEADER_SIZE);
+            BufferUtil.writeUB3(buffer, size);
+            buffer.put(packetId);
+            writeBody(buffer, isFirst, remain);
+            c.write(buffer);
+        } catch (java.nio.BufferOverflowException e1) {
+            //fixed issues #98 #1072
```

Comment:
does the catch logic meaningful, the code does not use c.allocate() in writeBigPackage() like in write() function